### PR TITLE
Iris release 1p11

### PR DESCRIPTION
**Pull request by bjlittle**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 
 os: osx
+osx_image: beta-xcode6.1
 
 env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```
Comment (bjlittle) [inline .travis.yml]: A fresh rerender seem to be still nuking `34` and `35` ...

Comment (bjlittle) [inline .travis.yml]: ... and this is my `.condarc`:
```
changeps1: 'no'

show_channel_urls: true

add_pip_as_python_dependency: false

channels:
  - conda-forge
  - defaults
  - scitools
```
... if it matters.

**Comment by bjlittle** [comment]
@ocefpaf and @jakirkham Any ideas ...

**Comment by ocefpaf** [comment]
Your condarc seems OK and to debug this will be quite annoying :-(

You can re-add the matrix manually, or wait until I get back on the 3rd and
I will rerender it with my hacked* conda-smithy ; -)

* I edited the package index manually to ensure the rerendering will always
be correct.

**Comment by ocefpaf** [comment]
Replaced by #14

**Pull request closed without merging**